spec/controllers/subscriptions_controller_spec.rb

```diff
@@ -8,77 +8,52 @@
   describe '#new' do
     subject { get :new, params: { search_criteria: { subject: 'english' } } }
```

Comment:
We used to have a subjects filter, but now we don't, so this should probably be a keyword, not a subject.